Building OpenBLAS with MinGW from msys2.org fails

I am trying to build OpenBLAS on Windows 10, strictly following the instruction in this [link](https://github.com/xianyi/OpenBLAS/wiki/How-to-use-OpenBLAS-in-Microsoft-Visual-Studio) under the section "Build OpenBLAS on Windows OS". The environment I'm using is Windows 10, Intel core i5-6200u, MinGW64 from http://www.msys2.org. The following is the full message shown in the MSYS2 MinGW 64-bit shell after rerunning the command again after the first error.

```
DESKTOP-DI9LHTR /d/Downloads/OpenBLAS/OpenBLAS-0.2.20
$ make BINARY=64 CC=gcc FC=gfortran
make[1]: Entering directory `/d/Downloads/OpenBLAS/OpenBLAS-0.2.20/interface'
ar  -ru ../libopenblas_haswellp-r0.2.20.a saxpy.obj sswap.obj scopy.obj sscal.obj sdot.obj sdsdot.obj dsdot.obj sasum.obj snrm2.obj smax.obj samax.obj ismax.obj isamax.obj smin.obj samin.obj ismin.obj isamin.obj srot.obj srotg.obj srotm.obj srotmg.obj saxpby.obj cblas_isamax.obj cblas_sasum.obj cblas_saxpy.obj cblas_scopy.obj cblas_sdot.obj cblas_sdsdot.obj cblas_dsdot.obj cblas_srot.obj cblas_srotg.obj cblas_srotm.obj cblas_srotmg.obj cblas_sscal.obj cblas_sswap.obj cblas_snrm2.obj cblas_saxpby.obj sgemv.obj sger.obj strsv.obj strmv.obj ssymv.obj ssyr.obj ssyr2.obj sgbmv.obj ssbmv.obj sspmv.obj sspr.obj sspr2.obj stbsv.obj stbmv.obj stpsv.obj stpmv.obj cblas_sgemv.obj cblas_sger.obj cblas_ssymv.obj cblas_strmv.obj cblas_strsv.obj cblas_ssyr.obj cblas_ssyr2.obj cblas_sgbmv.obj cblas_ssbmv.obj cblas_sspmv.obj cblas_sspr.obj cblas_sspr2.obj cblas_stbmv.obj cblas_stbsv.obj cblas_stpmv.obj cblas_stpsv.obj sgemm.obj ssymm.obj strmm.obj strsm.obj ssyrk.obj ssyr2k.obj somatcopy.obj simatcopy.obj sgeadd.obj cblas_sgemm.obj cblas_ssymm.obj cblas_strmm.obj cblas_strsm.obj cblas_ssyrk.obj cblas_ssyr2k.obj cblas_somatcopy.obj cblas_simatcopy.obj cblas_sgeadd.obj sgetrf.obj sgetrs.obj spotrf.obj sgetf2.obj spotf2.obj slaswp.obj sgesv.obj slauu2.obj slauum.obj strti2.obj strtri.obj daxpy.obj dswap.obj dcopy.obj dscal.obj ddot.obj dasum.obj dnrm2.obj dmax.obj damax.obj idmax.obj idamax.obj dmin.obj damin.obj idmin.obj idamin.obj drot.obj drotg.obj drotm.obj drotmg.obj daxpby.obj cblas_idamax.obj cblas_dasum.obj cblas_daxpy.obj cblas_dcopy.obj cblas_ddot.obj cblas_drot.obj cblas_drotg.obj cblas_drotm.obj cblas_drotmg.obj cblas_dscal.obj cblas_dswap.obj cblas_dnrm2.obj cblas_daxpby.obj dgemv.obj dger.obj dtrsv.obj dtrmv.obj dsymv.obj dsyr.obj dsyr2.obj dgbmv.obj dsbmv.obj dspmv.obj dspr.obj dspr2.obj dtbsv.obj dtbmv.obj dtpsv.obj dtpmv.obj cblas_dgemv.obj cblas_dger.obj cblas_dsymv.obj cblas_dtrmv.obj cblas_dtrsv.obj cblas_dsyr.obj cblas_dsyr2.obj cblas_dgbmv.obj cblas_dsbmv.obj cblas_dspmv.obj cblas_dspr.obj cblas_dspr2.obj cblas_dtbmv.obj cblas_dtbsv.obj cblas_dtpmv.obj cblas_dtpsv.obj dgemm.obj dsymm.obj dtrmm.obj dtrsm.obj dsyrk.obj dsyr2k.obj domatcopy.obj dimatcopy.obj dgeadd.obj cblas_dgemm.obj cblas_dsymm.obj cblas_dtrmm.obj cblas_dtrsm.obj cblas_dsyrk.obj cblas_dsyr2k.obj cblas_domatcopy.obj cblas_dimatcopy.obj cblas_dgeadd.obj dgetrf.obj dgetrs.obj dpotrf.obj dgetf2.obj dpotf2.obj dlaswp.obj dgesv.obj dlauu2.obj dlauum.obj dtrti2.obj dtrtri.obj caxpy.obj caxpyc.obj cswap.obj ccopy.obj cscal.obj csscal.obj cdotc.obj cdotu.obj scasum.obj scnrm2.obj scamax.obj icamax.obj scamin.obj icamin.obj csrot.obj crotg.obj caxpby.obj cblas_icamax.obj cblas_scasum.obj cblas_caxpy.obj cblas_ccopy.obj cblas_cdotc.obj cblas_cdotu.obj cblas_cdotc_sub.obj cblas_cdotu_sub.obj cblas_cscal.obj cblas_csscal.obj cblas_cswap.obj cblas_scnrm2.obj cblas_caxpby.obj cgemv.obj cgeru.obj cgerc.obj ctrsv.obj ctrmv.obj csyr2.obj cgbmv.obj csbmv.obj cspr2.obj ctbsv.obj ctbmv.obj ctpsv.obj ctpmv.obj chemv.obj chbmv.obj cher.obj cher2.obj chpmv.obj chpr.obj chpr2.obj cblas_cgemv.obj cblas_cgerc.obj cblas_cgeru.obj cblas_cgbmv.obj cblas_chbmv.obj cblas_chemv.obj cblas_cher.obj cblas_cher2.obj cblas_chpmv.obj cblas_chpr.obj cblas_chpr2.obj cblas_ctbmv.obj cblas_ctbsv.obj cblas_ctpmv.obj cblas_ctpsv.obj cblas_ctrmv.obj cblas_ctrsv.obj cgemm.obj csymm.obj ctrmm.obj ctrsm.obj csyrk.obj csyr2k.obj chemm.obj cherk.obj cher2k.obj comatcopy.obj cimatcopy.obj cgeadd.obj cgemm3m.obj cblas_cgemm.obj cblas_csymm.obj cblas_ctrmm.obj cblas_ctrsm.obj cblas_csyrk.obj cblas_csyr2k.obj cblas_chemm.obj cblas_cherk.obj cblas_cher2k.obj cblas_comatcopy.obj cblas_cimatcopy.obj cblas_cgeadd.obj cblas_xerbla.obj cblas_cgemm3m.obj cgetrf.obj cgetrs.obj cpotrf.obj cgetf2.obj cpotf2.obj claswp.obj cgesv.obj clauu2.obj clauum.obj ctrti2.obj ctrtri.obj zaxpy.obj zaxpyc.obj zswap.obj zcopy.obj zscal.obj zdscal.obj zdotc.obj zdotu.obj dzasum.obj dznrm2.obj dzamax.obj izamax.obj dzamin.obj izamin.obj zdrot.obj zrotg.obj zaxpby.obj cblas_izamax.obj cblas_dzasum.obj cblas_zaxpy.obj cblas_zcopy.obj cblas_zdotc.obj cblas_zdotu.obj cblas_zdotc_sub.obj cblas_zdotu_sub.obj cblas_zscal.obj cblas_zdscal.obj cblas_zswap.obj cblas_dznrm2.obj cblas_zaxpby.obj zgemv.obj zgeru.obj zgerc.obj ztrsv.obj ztrmv.obj zsyr2.obj zgbmv.obj zsbmv.obj zspr2.obj ztbsv.obj ztbmv.obj ztpsv.obj ztpmv.obj zhemv.obj zhbmv.obj zher.obj zher2.obj zhpmv.obj zhpr.obj zhpr2.obj cblas_zgemv.obj cblas_zgerc.obj cblas_zgeru.obj cblas_zgbmv.obj cblas_zhbmv.obj cblas_zhemv.obj cblas_zher.obj cblas_zher2.obj cblas_zhpmv.obj cblas_zhpr.obj cblas_zhpr2.obj cblas_ztbmv.obj cblas_ztbsv.obj cblas_ztpmv.obj cblas_ztpsv.obj cblas_ztrmv.obj cblas_ztrsv.obj zgemm.obj zsymm.obj ztrmm.obj ztrsm.obj zsyrk.obj zsyr2k.obj zhemm.obj zherk.obj zher2k.obj zomatcopy.obj zimatcopy.obj zgeadd.obj zgemm3m.obj cblas_zgemm.obj cblas_zsymm.obj cblas_ztrmm.obj cblas_ztrsm.obj cblas_zsyrk.obj cblas_zsyr2k.obj cblas_zhemm.obj cblas_zherk.obj cblas_zher2k.obj cblas_zomatcopy.obj cblas_zimatcopy.obj cblas_zgeadd.obj cblas_zgemm3m.obj zgetrf.obj zgetrs.obj zpotrf.obj zgetf2.obj zpotf2.obj zlaswp.obj zgesv.obj zlauu2.obj zlauum.obj ztrti2.obj ztrtri.obj
make[1]: Leaving directory `/d/Downloads/OpenBLAS/OpenBLAS-0.2.20/interface'
make[1]: Entering directory `/d/Downloads/OpenBLAS/OpenBLAS-0.2.20/driver/level2'
ar  -ru ../../libopenblas_haswellp-r0.2.20.a sgbmv_n.obj sgbmv_t.obj ssbmv_U.obj ssbmv_L.obj sspmv_U.obj sspmv_L.obj sspr_U.obj sspr_L.obj sspr2_U.obj sspr2_L.obj ssyr_U.obj ssyr_L.obj ssyr2_U.obj ssyr2_L.obj stbmv_NUU.obj stbmv_NUN.obj stbmv_NLU.obj stbmv_NLN.obj stbmv_TUU.obj stbmv_TUN.obj stbmv_TLU.obj stbmv_TLN.obj stbsv_NUU.obj stbsv_NUN.obj stbsv_NLU.obj stbsv_NLN.obj stbsv_TUU.obj stbsv_TUN.obj stbsv_TLU.obj stbsv_TLN.obj stpmv_NUU.obj stpmv_NUN.obj stpmv_NLU.obj stpmv_NLN.obj stpmv_TUU.obj stpmv_TUN.obj stpmv_TLU.obj stpmv_TLN.obj stpsv_NUU.obj stpsv_NUN.obj stpsv_NLU.obj stpsv_NLN.obj stpsv_TUU.obj stpsv_TUN.obj stpsv_TLU.obj stpsv_TLN.obj strmv_NUU.obj strmv_NUN.obj strmv_NLU.obj strmv_NLN.obj strmv_TUU.obj strmv_TUN.obj strmv_TLU.obj strmv_TLN.obj strsv_NUU.obj strsv_NUN.obj strsv_NLU.obj strsv_NLN.obj strsv_TUU.obj strsv_TUN.obj strsv_TLU.obj strsv_TLN.obj sgemv_thread_n.obj sgemv_thread_t.obj sger_thread.obj ssymv_thread_U.obj ssymv_thread_L.obj ssyr_thread_U.obj ssyr_thread_L.obj ssyr2_thread_U.obj ssyr2_thread_L.obj sspr_thread_U.obj sspr_thread_L.obj sspr2_thread_U.obj sspr2_thread_L.obj strmv_thread_NUU.obj strmv_thread_NUN.obj strmv_thread_NLU.obj strmv_thread_NLN.obj strmv_thread_TUU.obj strmv_thread_TUN.obj strmv_thread_TLU.obj strmv_thread_TLN.obj sspmv_thread_U.obj sspmv_thread_L.obj stpmv_thread_NUU.obj stpmv_thread_NUN.obj stpmv_thread_NLU.obj stpmv_thread_NLN.obj stpmv_thread_TUU.obj stpmv_thread_TUN.obj stpmv_thread_TLU.obj stpmv_thread_TLN.obj sgbmv_thread_n.obj sgbmv_thread_t.obj ssbmv_thread_U.obj ssbmv_thread_L.obj stbmv_thread_NUU.obj stbmv_thread_NUN.obj stbmv_thread_NLU.obj stbmv_thread_NLN.obj stbmv_thread_TUU.obj stbmv_thread_TUN.obj stbmv_thread_TLU.obj stbmv_thread_TLN.obj dgbmv_n.obj dgbmv_t.obj dsbmv_U.obj dsbmv_L.obj dspmv_U.obj dspmv_L.obj dspr_U.obj dspr_L.obj dspr2_U.obj dspr2_L.obj dsyr_U.obj dsyr_L.obj dsyr2_U.obj dsyr2_L.obj dtbmv_NUU.obj dtbmv_NUN.obj dtbmv_NLU.obj dtbmv_NLN.obj dtbmv_TUU.obj dtbmv_TUN.obj dtbmv_TLU.obj dtbmv_TLN.obj dtbsv_NUU.obj dtbsv_NUN.obj dtbsv_NLU.obj dtbsv_NLN.obj dtbsv_TUU.obj dtbsv_TUN.obj dtbsv_TLU.obj dtbsv_TLN.obj dtpmv_NUU.obj dtpmv_NUN.obj dtpmv_NLU.obj dtpmv_NLN.obj dtpmv_TUU.obj dtpmv_TUN.obj dtpmv_TLU.obj dtpmv_TLN.obj dtpsv_NUU.obj dtpsv_NUN.obj dtpsv_NLU.obj dtpsv_NLN.obj dtpsv_TUU.obj dtpsv_TUN.obj dtpsv_TLU.obj dtpsv_TLN.obj dtrmv_NUU.obj dtrmv_NUN.obj dtrmv_NLU.obj dtrmv_NLN.obj dtrmv_TUU.obj dtrmv_TUN.obj dtrmv_TLU.obj dtrmv_TLN.obj dtrsv_NUU.obj dtrsv_NUN.obj dtrsv_NLU.obj dtrsv_NLN.obj dtrsv_TUU.obj dtrsv_TUN.obj dtrsv_TLU.obj dtrsv_TLN.obj dgemv_thread_n.obj dgemv_thread_t.obj dger_thread.obj dsymv_thread_U.obj dsymv_thread_L.obj dsyr_thread_U.obj dsyr_thread_L.obj dsyr2_thread_U.obj dsyr2_thread_L.obj dspr_thread_U.obj dspr_thread_L.obj dspr2_thread_U.obj dspr2_thread_L.obj dtrmv_thread_NUU.obj dtrmv_thread_NUN.obj dtrmv_thread_NLU.obj dtrmv_thread_NLN.obj dtrmv_thread_TUU.obj dtrmv_thread_TUN.obj dtrmv_thread_TLU.obj dtrmv_thread_TLN.obj dspmv_thread_U.obj dspmv_thread_L.obj dtpmv_thread_NUU.obj dtpmv_thread_NUN.obj dtpmv_thread_NLU.obj dtpmv_thread_NLN.obj dtpmv_thread_TUU.obj dtpmv_thread_TUN.obj dtpmv_thread_TLU.obj dtpmv_thread_TLN.obj dgbmv_thread_n.obj dgbmv_thread_t.obj dsbmv_thread_U.obj dsbmv_thread_L.obj dtbmv_thread_NUU.obj dtbmv_thread_NUN.obj dtbmv_thread_NLU.obj dtbmv_thread_NLN.obj dtbmv_thread_TUU.obj dtbmv_thread_TUN.obj dtbmv_thread_TLU.obj dtbmv_thread_TLN.obj cgbmv_n.obj cgbmv_t.obj cgbmv_r.obj cgbmv_c.obj cgbmv_o.obj cgbmv_u.obj cgbmv_s.obj cgbmv_d.obj chbmv_U.obj chbmv_L.obj chbmv_V.obj chbmv_M.obj cher_U.obj cher_L.obj cher_V.obj cher_M.obj cher2_U.obj cher2_L.obj cher2_V.obj cher2_M.obj chpmv_U.obj chpmv_L.obj chpmv_V.obj chpmv_M.obj chpr_U.obj chpr_L.obj chpr_V.obj chpr_M.obj chpr2_U.obj chpr2_L.obj chpr2_V.obj chpr2_M.obj csbmv_U.obj csbmv_L.obj cspmv_U.obj cspmv_L.obj cspr_U.obj cspr_L.obj cspr2_U.obj cspr2_L.obj csyr_U.obj csyr_L.obj csyr2_U.obj csyr2_L.obj ctbmv_NUU.obj ctbmv_NUN.obj ctbmv_NLU.obj ctbmv_NLN.obj ctbmv_TUU.obj ctbmv_TUN.obj ctbmv_TLU.obj ctbmv_TLN.obj ctbmv_RUU.obj ctbmv_RUN.obj ctbmv_RLU.obj ctbmv_RLN.obj ctbmv_CUU.obj ctbmv_CUN.obj ctbmv_CLU.obj ctbmv_CLN.obj ctbsv_NUU.obj ctbsv_NUN.obj ctbsv_NLU.obj ctbsv_NLN.obj ctbsv_TUU.obj ctbsv_TUN.obj ctbsv_TLU.obj ctbsv_TLN.obj ctbsv_RUU.obj ctbsv_RUN.obj ctbsv_RLU.obj ctbsv_RLN.obj ctbsv_CUU.obj ctbsv_CUN.obj ctbsv_CLU.obj ctbsv_CLN.obj ctpmv_NUU.obj ctpmv_NUN.obj ctpmv_NLU.obj ctpmv_NLN.obj ctpmv_TUU.obj ctpmv_TUN.obj ctpmv_TLU.obj ctpmv_TLN.obj ctpmv_RUU.obj ctpmv_RUN.obj ctpmv_RLU.obj ctpmv_RLN.obj ctpmv_CUU.obj ctpmv_CUN.obj ctpmv_CLU.obj ctpmv_CLN.obj ctpsv_NUU.obj ctpsv_NUN.obj ctpsv_NLU.obj ctpsv_NLN.obj ctpsv_TUU.obj ctpsv_TUN.obj ctpsv_TLU.obj ctpsv_TLN.obj ctpsv_RUU.obj ctpsv_RUN.obj ctpsv_RLU.obj ctpsv_RLN.obj ctpsv_CUU.obj ctpsv_CUN.obj ctpsv_CLU.obj ctpsv_CLN.obj ctrmv_NUU.obj ctrmv_NUN.obj ctrmv_NLU.obj ctrmv_NLN.obj ctrmv_TUU.obj ctrmv_TUN.obj ctrmv_TLU.obj ctrmv_TLN.obj ctrmv_RUU.obj ctrmv_RUN.obj ctrmv_RLU.obj ctrmv_RLN.obj ctrmv_CUU.obj ctrmv_CUN.obj ctrmv_CLU.obj ctrmv_CLN.obj ctrsv_NUU.obj ctrsv_NUN.obj ctrsv_NLU.obj ctrsv_NLN.obj ctrsv_TUU.obj ctrsv_TUN.obj ctrsv_TLU.obj ctrsv_TLN.obj ctrsv_RUU.obj ctrsv_RUN.obj ctrsv_RLU.obj ctrsv_RLN.obj ctrsv_CUU.obj ctrsv_CUN.obj ctrsv_CLU.obj ctrsv_CLN.obj cgemv_thread_n.obj cgemv_thread_t.obj cgemv_thread_r.obj cgemv_thread_c.obj cgemv_thread_o.obj cgemv_thread_u.obj cgemv_thread_s.obj cgemv_thread_d.obj cger_thread_U.obj cger_thread_C.obj cger_thread_V.obj cger_thread_D.obj csymv_thread_U.obj csymv_thread_L.obj chemv_thread_U.obj chemv_thread_L.obj chemv_thread_V.obj chemv_thread_M.obj csyr_thread_U.obj csyr_thread_L.obj cher_thread_U.obj cher_thread_L.obj cher_thread_V.obj cher_thread_M.obj csyr2_thread_U.obj csyr2_thread_L.obj cher2_thread_U.obj cher2_thread_L.obj cher2_thread_V.obj cher2_thread_M.obj cspr_thread_U.obj cspr_thread_L.obj chpr_thread_U.obj chpr_thread_L.obj chpr_thread_V.obj chpr_thread_M.obj cspr2_thread_U.obj cspr2_thread_L.obj chpr2_thread_U.obj chpr2_thread_L.obj chpr2_thread_V.obj chpr2_thread_M.obj ctrmv_thread_NUU.obj ctrmv_thread_NUN.obj ctrmv_thread_NLU.obj ctrmv_thread_NLN.obj ctrmv_thread_TUU.obj ctrmv_thread_TUN.obj ctrmv_thread_TLU.obj ctrmv_thread_TLN.obj ctrmv_thread_RUU.obj ctrmv_thread_RUN.obj ctrmv_thread_RLU.obj ctrmv_thread_RLN.obj ctrmv_thread_CUU.obj ctrmv_thread_CUN.obj ctrmv_thread_CLU.obj ctrmv_thread_CLN.obj cspmv_thread_U.obj cspmv_thread_L.obj chpmv_thread_U.obj chpmv_thread_L.obj chpmv_thread_V.obj chpmv_thread_M.obj ctpmv_thread_NUU.obj ctpmv_thread_NUN.obj ctpmv_thread_NLU.obj ctpmv_thread_NLN.obj ctpmv_thread_TUU.obj ctpmv_thread_TUN.obj ctpmv_thread_TLU.obj ctpmv_thread_TLN.obj ctpmv_thread_RUU.obj ctpmv_thread_RUN.obj ctpmv_thread_RLU.obj ctpmv_thread_RLN.obj ctpmv_thread_CUU.obj ctpmv_thread_CUN.obj ctpmv_thread_CLU.obj ctpmv_thread_CLN.obj cgbmv_thread_n.obj cgbmv_thread_t.obj cgbmv_thread_r.obj cgbmv_thread_c.obj cgbmv_thread_o.obj cgbmv_thread_u.obj cgbmv_thread_s.obj cgbmv_thread_d.obj csbmv_thread_U.obj csbmv_thread_L.obj chbmv_thread_U.obj chbmv_thread_L.obj chbmv_thread_V.obj chbmv_thread_M.obj ctbmv_thread_NUU.obj ctbmv_thread_NUN.obj ctbmv_thread_NLU.obj ctbmv_thread_NLN.obj ctbmv_thread_TUU.obj ctbmv_thread_TUN.obj ctbmv_thread_TLU.obj ctbmv_thread_TLN.obj ctbmv_thread_RUU.obj ctbmv_thread_RUN.obj ctbmv_thread_RLU.obj ctbmv_thread_RLN.obj ctbmv_thread_CUU.obj ctbmv_thread_CUN.obj ctbmv_thread_CLU.obj ctbmv_thread_CLN.obj zgbmv_n.obj zgbmv_t.obj zgbmv_r.obj zgbmv_c.obj zgbmv_o.obj zgbmv_u.obj zgbmv_s.obj zgbmv_d.obj zhbmv_U.obj zhbmv_L.obj zhbmv_V.obj zhbmv_M.obj zher_U.obj zher_L.obj zher_V.obj zher_M.obj zher2_U.obj zher2_L.obj zher2_V.obj zher2_M.obj zhpmv_U.obj zhpmv_L.obj zhpmv_V.obj zhpmv_M.obj zhpr_U.obj zhpr_L.obj zhpr_V.obj zhpr_M.obj zhpr2_U.obj zhpr2_L.obj zhpr2_V.obj zhpr2_M.obj zsbmv_U.obj zsbmv_L.obj zspmv_U.obj zspmv_L.obj zspr_U.obj zspr_L.obj zspr2_U.obj zspr2_L.obj zsyr_U.obj zsyr_L.obj zsyr2_U.obj zsyr2_L.obj ztbmv_NUU.obj ztbmv_NUN.obj ztbmv_NLU.obj ztbmv_NLN.obj ztbmv_TUU.obj ztbmv_TUN.obj ztbmv_TLU.obj ztbmv_TLN.obj ztbmv_RUU.obj ztbmv_RUN.obj ztbmv_RLU.obj ztbmv_RLN.obj ztbmv_CUU.obj ztbmv_CUN.obj ztbmv_CLU.obj ztbmv_CLN.obj ztbsv_NUU.obj ztbsv_NUN.obj ztbsv_NLU.obj ztbsv_NLN.obj ztbsv_TUU.obj ztbsv_TUN.obj ztbsv_TLU.obj ztbsv_TLN.obj ztbsv_RUU.obj ztbsv_RUN.obj ztbsv_RLU.obj ztbsv_RLN.obj ztbsv_CUU.obj ztbsv_CUN.obj ztbsv_CLU.obj ztbsv_CLN.obj ztpmv_NUU.obj ztpmv_NUN.obj ztpmv_NLU.obj ztpmv_NLN.obj ztpmv_TUU.obj ztpmv_TUN.obj ztpmv_TLU.obj ztpmv_TLN.obj ztpmv_RUU.obj ztpmv_RUN.obj ztpmv_RLU.obj ztpmv_RLN.obj ztpmv_CUU.obj ztpmv_CUN.obj ztpmv_CLU.obj ztpmv_CLN.obj ztpsv_NUU.obj ztpsv_NUN.obj ztpsv_NLU.obj ztpsv_NLN.obj ztpsv_TUU.obj ztpsv_TUN.obj ztpsv_TLU.obj ztpsv_TLN.obj ztpsv_RUU.obj ztpsv_RUN.obj ztpsv_RLU.obj ztpsv_RLN.obj ztpsv_CUU.obj ztpsv_CUN.obj ztpsv_CLU.obj ztpsv_CLN.obj ztrmv_NUU.obj ztrmv_NUN.obj ztrmv_NLU.obj ztrmv_NLN.obj ztrmv_TUU.obj ztrmv_TUN.obj ztrmv_TLU.obj ztrmv_TLN.obj ztrmv_RUU.obj ztrmv_RUN.obj ztrmv_RLU.obj ztrmv_RLN.obj ztrmv_CUU.obj ztrmv_CUN.obj ztrmv_CLU.obj ztrmv_CLN.obj ztrsv_NUU.obj ztrsv_NUN.obj ztrsv_NLU.obj ztrsv_NLN.obj ztrsv_TUU.obj ztrsv_TUN.obj ztrsv_TLU.obj ztrsv_TLN.obj ztrsv_RUU.obj ztrsv_RUN.obj ztrsv_RLU.obj ztrsv_RLN.obj ztrsv_CUU.obj ztrsv_CUN.obj ztrsv_CLU.obj ztrsv_CLN.obj zgemv_thread_n.obj zgemv_thread_t.obj zgemv_thread_r.obj zgemv_thread_c.obj zgemv_thread_o.obj zgemv_thread_u.obj zgemv_thread_s.obj zgemv_thread_d.obj zger_thread_U.obj zger_thread_C.obj zger_thread_V.obj zger_thread_D.obj zsymv_thread_U.obj zsymv_thread_L.obj zhemv_thread_U.obj zhemv_thread_L.obj zhemv_thread_V.obj zhemv_thread_M.obj zsyr_thread_U.obj zsyr_thread_L.obj zher_thread_U.obj zher_thread_L.obj zher_thread_V.obj zher_thread_M.obj zsyr2_thread_U.obj zsyr2_thread_L.obj zher2_thread_U.obj zher2_thread_L.obj zher2_thread_V.obj zher2_thread_M.obj zspr_thread_U.obj zspr_thread_L.obj zhpr_thread_U.obj zhpr_thread_L.obj zhpr_thread_V.obj zhpr_thread_M.obj zspr2_thread_U.obj zspr2_thread_L.obj zhpr2_thread_U.obj zhpr2_thread_L.obj zhpr2_thread_V.obj zhpr2_thread_M.obj ztrmv_thread_NUU.obj ztrmv_thread_NUN.obj ztrmv_thread_NLU.obj ztrmv_thread_NLN.obj ztrmv_thread_TUU.obj ztrmv_thread_TUN.obj ztrmv_thread_TLU.obj ztrmv_thread_TLN.obj ztrmv_thread_RUU.obj ztrmv_thread_RUN.obj ztrmv_thread_RLU.obj ztrmv_thread_RLN.obj ztrmv_thread_CUU.obj ztrmv_thread_CUN.obj ztrmv_thread_CLU.obj ztrmv_thread_CLN.obj zspmv_thread_U.obj zspmv_thread_L.obj zhpmv_thread_U.obj zhpmv_thread_L.obj zhpmv_thread_V.obj zhpmv_thread_M.obj ztpmv_thread_NUU.obj ztpmv_thread_NUN.obj ztpmv_thread_NLU.obj ztpmv_thread_NLN.obj ztpmv_thread_TUU.obj ztpmv_thread_TUN.obj ztpmv_thread_TLU.obj ztpmv_thread_TLN.obj ztpmv_thread_RUU.obj ztpmv_thread_RUN.obj ztpmv_thread_RLU.obj ztpmv_thread_RLN.obj ztpmv_thread_CUU.obj ztpmv_thread_CUN.obj ztpmv_thread_CLU.obj ztpmv_thread_CLN.obj zgbmv_thread_n.obj zgbmv_thread_t.obj zgbmv_thread_r.obj zgbmv_thread_c.obj zgbmv_thread_o.obj zgbmv_thread_u.obj zgbmv_thread_s.obj zgbmv_thread_d.obj zsbmv_thread_U.obj zsbmv_thread_L.obj zhbmv_thread_U.obj zhbmv_thread_L.obj zhbmv_thread_V.obj zhbmv_thread_M.obj ztbmv_thread_NUU.obj ztbmv_thread_NUN.obj ztbmv_thread_NLU.obj ztbmv_thread_NLN.obj ztbmv_thread_TUU.obj ztbmv_thread_TUN.obj ztbmv_thread_TLU.obj ztbmv_thread_TLN.obj ztbmv_thread_RUU.obj ztbmv_thread_RUN.obj ztbmv_thread_RLU.obj ztbmv_thread_RLN.obj ztbmv_thread_CUU.obj ztbmv_thread_CUN.obj ztbmv_thread_CLU.obj ztbmv_thread_CLN.obj
make[1]: Leaving directory `/d/Downloads/OpenBLAS/OpenBLAS-0.2.20/driver/level2'
make[1]: Entering directory `/d/Downloads/OpenBLAS/OpenBLAS-0.2.20/driver/level3'
gcc -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=cgemm_cr -DASMFNAME=cgemm_cr_ -DNAME=cgemm_cr_ -DCNAME=cgemm_cr -DCHAR_NAME=\"cgemm_cr_\" -DCHAR_CNAME=\"cgemm_cr\" -DNO_AFFINITY -I../.. -UDOUBLE  -DCOMPLEX  -c -UDOUBLE -DCOMPLEX -DCR gemm.c -o cgemm_cr.obj
<command-line>:0:4: error: expected identifier or '(' before numeric constant
make[1]: *** [cgemm_cr.obj] Error 1
make[1]: Leaving directory `/d/Downloads/OpenBLAS/OpenBLAS-0.2.20/driver/level3'
make: *** [libs] Error 1

DESKTOP-DI9LHTR /d/Downloads/OpenBLAS/OpenBLAS-0.2.20
$ gcc -v
Using built-in specs.
COLLECT_GCC=C:\msys64\mingw64\bin\gcc.exe
COLLECT_LTO_WRAPPER=C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/7.3.0/lto-wrapper.exe
Target: x86_64-w64-mingw32
Configured with: ../gcc-7.3.0/configure --prefix=/mingw64 --with-local-prefix=/mingw64/local --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 --with-native-system-header-dir=/mingw64/x86_64-w64-mingw32/include --libexecdir=/mingw64/lib --enable-bootstrap --with-arch=x86-64 --with-tune=generic --enable-languages=c,lto,c++,objc,obj-c++,fortran,ada --enable-shared --enable-static --enable-libatomic --enable-threads=posix --enable-graphite --enable-fully-dynamic-string --enable-libstdcxx-time=yes --enable-libstdcxx-filesystem-ts=yes --disable-libstdcxx-pch --disable-libstdcxx-debug --disable-isl-version-check --enable-lto --enable-libgomp --disable-multilib --enable-checking=release --disable-rpath --disable-win32-registry --disable-nls --disable-werror --disable-symvers --with-libiconv --with-system-zlib --with-gmp=/mingw64 --with-mpfr=/mingw64 --with-mpc=/mingw64 --with-isl=/mingw64 --with-pkgversion='Rev1, Built by MSYS2 project' --with-bugurl=https://sourceforge.net/projects/msys2 --with-gnu-as --with-gnu-ld
Thread model: posix
gcc version 7.3.0 (Rev1, Built by MSYS2 project)

DESKTOP-DI9LHTR /d/Downloads/OpenBLAS/OpenBLAS-0.2.20
$ gfortran -v
Using built-in specs.
COLLECT_GCC=C:\msys64\mingw64\bin\gfortran.exe
COLLECT_LTO_WRAPPER=C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/7.3.0/lto-wrapper.exe
Target: x86_64-w64-mingw32
Configured with: ../gcc-7.3.0/configure --prefix=/mingw64 --with-local-prefix=/mingw64/local --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 --with-native-system-header-dir=/mingw64/x86_64-w64-mingw32/include --libexecdir=/mingw64/lib --enable-bootstrap --with-arch=x86-64 --with-tune=generic --enable-languages=c,lto,c++,objc,obj-c++,fortran,ada --enable-shared --enable-static --enable-libatomic --enable-threads=posix --enable-graphite --enable-fully-dynamic-string --enable-libstdcxx-time=yes --enable-libstdcxx-filesystem-ts=yes --disable-libstdcxx-pch --disable-libstdcxx-debug --disable-isl-version-check --enable-lto --enable-libgomp --disable-multilib --enable-checking=release --disable-rpath --disable-win32-registry --disable-nls --disable-werror --disable-symvers --with-libiconv --with-system-zlib --with-gmp=/mingw64 --with-mpfr=/mingw64 --with-mpc=/mingw64 --with-isl=/mingw64 --with-pkgversion='Rev1, Built by MSYS2 project' --with-bugurl=https://sourceforge.net/projects/msys2 --with-gnu-as --with-gnu-ld
Thread model: posix
gcc version 7.3.0 (Rev1, Built by MSYS2 project)
```

What could be the source of this error?
`<command-line>:0:4: error: expected identifier or '(' before numeric constant`

Still have the same error. Also, I haven't mentioned that it had 2 warnings during the compilation (if it matters).
```
$ ./quickbuild.win64
 Please read https://github.com/xianyi/OpenBLAS/wiki/How-to-use-OpenBLAS-in-Microsoft-Visual-Studio
ln: failed to create symbolic link 'libopenblas.a': No such file or directory
make: [Makefile:138: libs] Error 1 (ignored)
make[1]: Entering directory '/d/Downloads/OpenBLAS/OpenBLAS-develop/interface'
gcc -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=saxpy -DASMFNAME=saxpy_ -DNAME=saxpy_ -DCNAME=saxpy -DCHAR_NAME=\"saxpy_\" -DCHAR_CNAME=\"saxpy\" -DNO_AFFINITY -I.. -I. -UDOUBLE  -UCOMPLEX -c axpy.c -o saxpy.obj
...
...
...
gcc -c -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=slaswp -DASMFNAME=slaswp_ -DNAME=slaswp_ -DCNAME=slaswp -DCHAR_NAME=\"slaswp_\" -DCHAR_CNAME=\"slaswp\" -DNO_AFFINITY -I.. -I. -UDOUBLE  -UCOMPLEX lapack/laswp.c -o slaswp.obj
lapack/laswp.c: In function 'slaswp_':
lapack/laswp.c:100:8: warning: passing argument 12 of 'blas_level1_thread' from incompatible pointer type [-Wincompatible-pointer-types]
        laswp[flag], nthreads);
        ^~~~~
In file included from ../common.h:777:0,
                 from lapack/laswp.c:40:
../common_thread.h:177:5: note: expected 'int (*)()' but argument is of type 'int (*)(BLASLONG,  BLASLONG,  BLASLONG,  float,  float *, BLASLONG,  float *, BLASLONG,  blasint *, BLASLONG) {aka int (*)(long long int,  long long int,  long long int,  float,  float *, long long int,  float *, long long int,  int *, long long int)}'
 int blas_level1_thread(int mode, BLASLONG m, BLASLONG n, BLASLONG k, void *alpha,
     ^~~~~~~~~~~~~~~~~~
gcc -c -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=sgesv -DASMFNAME=sgesv_ -DNAME=sgesv_ -DCNAME=sgesv -DCHAR_NAME=\"sgesv_\" -DCHAR_CNAME=\"sgesv\" -DNO_AFFINITY -I.. -I. -UDOUBLE  -UCOMPLEX lapack/gesv.c -o sgesv.obj
...
...
...
gcc -c -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=claswp -DASMFNAME=claswp_ -DNAME=claswp_ -DCNAME=claswp -DCHAR_NAME=\"claswp_\" -DCHAR_CNAME=\"claswp\" -DNO_AFFINITY -I.. -I. -UDOUBLE  -DCOMPLEX lapack/zlaswp.c -o claswp.obj
lapack/zlaswp.c: In function 'claswp_':
lapack/zlaswp.c:99:80: warning: passing argument 12 of 'blas_level1_thread' from incompatible pointer type [-Wincompatible-pointer-types]
   blas_level1_thread(mode, n, k1, k2, dummyalpha, a, lda, NULL, 0, ipiv, incx, laswp[flag], nthreads);
                                                                                ^~~~~
In file included from ../common.h:777:0,
                 from lapack/zlaswp.c:40:
../common_thread.h:177:5: note: expected 'int (*)()' but argument is of type 'int (*)(BLASLONG,  BLASLONG,  BLASLONG,  float,  float,  float *, BLASLONG,  float *, BLASLONG,  blasint *, BLASLONG) {aka int (*)(long long int,  long long int,  long long int,  float,  float,  float *, long long int,  float *, long long int,  int *, long long int)}'
 int blas_level1_thread(int mode, BLASLONG m, BLASLONG n, BLASLONG k, void *alpha,
     ^~~~~~~~~~~~~~~~~~
gcc -c -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=cgesv -DASMFNAME=cgesv_ -DNAME=cgesv_ -DCNAME=cgesv -DCHAR_NAME=\"cgesv_\" -DCHAR_CNAME=\"cgesv\" -DNO_AFFINITY -I.. -I. -UDOUBLE  -DCOMPLEX lapack/gesv.c -o cgesv.obj
...
...
...
gcc -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=cgemm_tr -DASMFNAME=cgemm_tr_ -DNAME=cgemm_tr_ -DCNAME=cgemm_tr -DCHAR_NAME=\"cgemm_tr_\" -DCHAR_CNAME=\"cgemm_tr\" -DNO_AFFINITY -I../.. -UDOUBLE  -DCOMPLEX  -c -UDOUBLE -DCOMPLEX -DTR gemm.c -o cgemm_tr.obj
gcc -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=cgemm_cr -DASMFNAME=cgemm_cr_ -DNAME=cgemm_cr_ -DCNAME=cgemm_cr -DCHAR_NAME=\"cgemm_cr_\" -DCHAR_CNAME=\"cgemm_cr\" -DNO_AFFINITY -I../.. -UDOUBLE  -DCOMPLEX  -c -UDOUBLE -DCOMPLEX -DCR gemm.c -o cgemm_cr.obj
<command-line>:0:4: error: expected identifier or '(' before numeric constant
make[1]: *** [Makefile:365: cgemm_cr.obj] Error 1
make[1]: Leaving directory '/d/Downloads/OpenBLAS/OpenBLAS-develop/driver/level3'
make: *** [Makefile:139: libs] Error 1
```
I really have no idea, maybe the problem with the compiler? So basically, I installed MinGW from the above [site](http://www.msys2.org) and just installed mingw64, msys, gcc, and gfortran package using `pacman -S` command. Should I try with another compiler?

@martin-frbg So, I've tried to invoke mingw-gcc (7.3.0) on level3.c and got the following: 
`Desktop\OpenBLAS\driver\level3>gcc level3.c`
`level3.c:172:11: error: unknown type name 'blas_arg_t'`
` int CNAME(blas_arg_t *args, BLASLONG *range_m, BLASLONG *range_n,`

`level3.c:172:29: error: unknown type name 'BLASLONG'`
` int CNAME(blas_arg_t *args, BLASLONG *range_m, BLASLONG *range_n,`

`level3.c:172:48: error: unknown type name 'BLASLONG'`
` int CNAME(blas_arg_t *args, BLASLONG *range_m, BLASLONG *range_n,`

`level3.c:173:5: error: unknown type name 'XFLOAT'`
`     XFLOAT *sa, XFLOAT *sb, BLASLONG dummy){`

`level3.c:173:17: error: unknown type name 'XFLOAT'`
`     XFLOAT *sa, XFLOAT *sb, BLASLONG dummy){`

`level3.c:173:29: error: unknown type name 'BLASLONG'`
`     XFLOAT *sa, XFLOAT *sb, BLASLONG dummy){` 
**Update 1:** got the same errors with command line `gcc -m64 -I../.. -DCR ../../param.h gemm.c level3.c` 
**Update 2:** added `#include "common.h` to `level3.c`-file and run `gcc -O2 -m64 -I../..  -c -DCR level3.c -o lev.obj` and it has been successfully executed without any error. I have no idea what's wrong...

After that I decided to try direct execution of this single line 
`gcc -O2 -DMS_ABI -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=4 -DASMNAME=cgemm_cr -DASMFNAME=cgemm_cr_ -DNAME=cgemm_cr_ -DCNAME=cgemm_cr -DCHAR_NAME=\"cgemm_cr_\" -DCHAR_CNAME=\"cgemm_cr\" -DNO_AFFINITY -I../.. -UDOUBLE  -DCOMPLEX  -c -UDOUBLE -DCOMPLEX -DCR gemm.c -o cgemm_cr.obj` 
and it has been successfully executed without any error

--- a/driver/level3/Makefile
+++ b/driver/level3/Makefile
@@ -362,7 +362,7 @@ cgemm_ct.$(SUFFIX) : gemm.c level3.c  ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm_cr.$(SUFFIX) : gemm.c level3.c  ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm_cc.$(SUFFIX) : gemm.c level3.c  ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -410,7 +410,7 @@ zgemm_ct.$(SUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm_cr.$(SUFFIX) : gemm.c level3.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm_cc.$(SUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -458,7 +458,7 @@ xgemm_ct.$(SUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm_cr.$(SUFFIX) : gemm.c level3.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm_cc.$(SUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -558,7 +558,7 @@ cgemm_thread_ct.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm_thread_cr.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm_thread_cc.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -606,7 +606,7 @@ zgemm_thread_ct.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm_thread_cr.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm_thread_cc.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -654,7 +654,7 @@ xgemm_thread_ct.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm_thread_cr.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm_thread_cc.$(SUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -1821,7 +1821,7 @@ cgemm3m_ct.$(SUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm3m_cr.$(SUFFIX) : gemm3m.c gemm3m_level3.c
-	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm3m_cc.$(SUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(CFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -1869,7 +1869,7 @@ zgemm3m_ct.$(SUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm3m_cr.$(SUFFIX) : gemm3m.c gemm3m_level3.c
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm3m_cc.$(SUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -1917,7 +1917,7 @@ xgemm3m_ct.$(SUFFIX) : gemm3m.c gemm3m_level3.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm3m_cr.$(SUFFIX) : gemm3m.c gemm3m_level3.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm3m_cc.$(SUFFIX) : gemm3m.c gemm3m_level3.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -1974,7 +1974,7 @@ cgemm3m_thread_ct.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm3m_thread_cr.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm3m_thread_cc.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2022,7 +2022,7 @@ zgemm3m_thread_ct.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm3m_thread_cr.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm3m_thread_cc.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2070,7 +2070,7 @@ xgemm3m_thread_ct.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm3m_thread_cr.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
-	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm3m_thread_cc.$(SUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(CFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2731,7 +2731,7 @@ cgemm_ct.$(PSUFFIX) : gemm.c level3.c  ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm_cr.$(PSUFFIX) : gemm.c level3.c  ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm_cc.$(PSUFFIX) : gemm.c level3.c  ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2779,7 +2779,7 @@ zgemm_ct.$(PSUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm_cr.$(PSUFFIX) : gemm.c level3.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm_cc.$(PSUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2827,7 +2827,7 @@ xgemm_ct.$(PSUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm_cr.$(PSUFFIX) : gemm.c level3.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm_cc.$(PSUFFIX) : gemm.c level3.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2927,7 +2927,7 @@ cgemm_thread_ct.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm_thread_cr.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm_thread_cc.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -2975,7 +2975,7 @@ zgemm_thread_ct.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm_thread_cr.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm_thread_cc.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -3023,7 +3023,7 @@ xgemm_thread_ct.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm_thread_cr.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm_thread_cc.$(PSUFFIX) : gemm.c level3_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -4190,7 +4190,7 @@ cgemm3m_ct.$(PSUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm3m_cr.$(PSUFFIX) : gemm3m.c gemm3m_level3.c
-	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm3m_cc.$(PSUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(PFLAGS) $(BLOCKS) -c -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -4238,7 +4238,7 @@ zgemm3m_ct.$(PSUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm3m_cr.$(PSUFFIX) : gemm3m.c gemm3m_level3.c
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm3m_cc.$(PSUFFIX) : gemm3m.c gemm3m_level3.c
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -4286,7 +4286,7 @@ xgemm3m_ct.$(PSUFFIX) : gemm3m.c gemm3m_level3.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm3m_cr.$(PSUFFIX) : gemm3m.c gemm3m_level3.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm3m_cc.$(PSUFFIX) : gemm3m.c gemm3m_level3.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -4343,7 +4343,7 @@ cgemm3m_thread_ct.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 cgemm3m_thread_cr.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 cgemm3m_thread_cc.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -UDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -4391,7 +4391,7 @@ zgemm3m_thread_ct.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 zgemm3m_thread_cr.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 zgemm3m_thread_cc.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DDOUBLE -DCOMPLEX -DCC $< -o $(@F)
@@ -4439,7 +4439,7 @@ xgemm3m_thread_ct.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCT $< -o $(@F)
 
 xgemm3m_thread_cr.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
-	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR $< -o $(@F)
+	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCR=CR $< -o $(@F)
 
 xgemm3m_thread_cc.$(PSUFFIX) : gemm3m.c level3_gemm3m_thread.c ../../param.h
 	$(CC) $(PFLAGS) $(BLOCKS) -c -DTHREADED_LEVEL3 -DXDOUBLE -DCOMPLEX -DCC $< -o $(@F)

